shared/Microsoft.Extensions.CommandLineUtils.Sources/Microsoft.Extensions.CommandLineUtils.Sources.projitems

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Question: What is a `.projitems` file meant to represent? Why even more factoring?

Reply:
This is using the "Shared Project" feature in VS (which works as far back as VS 2013). It's just UI sugar for working with files that are compiled into multiple projects.